src/patternfly/components/Banner/banner.hbs

```diff
@@ -0,0 +1,6 @@
+<div class="pf-c-banner{{#if banner--modifier}} {{banner--modifier}}{{/if}}"
```

Comment:
Not sure if this needs a role?
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Banner_role

Reply:
Any thoughts? @mcoker

Reply:
@jessiehuff wdyt?

Reply:
A banner landmark role should be reserved for the site header content that is common across the site generally at the top of every page. So if these banners are expected to be the top banner then they should have the role, but otherwise they shouldn't. Looking through the [design doc](https://docs.google.com/document/d/1QZY9EcblZaIGFB6KHqeDtqWPHqxsJn5-VGJnowmrVT8/edit), it almost looks like this is used similar to an alert so I wonder if it should be treated similar with an aria-label. wdyt @jgiardino?

Reply:
The [documentation for `role="banner"`](https://www.w3.org/TR/wai-aria-practices-1.1/#aria_lh_banner) is a bit confusing. After reading the following article, I'm inclined to say that we would not use `role="banner"` for this component:
https://www.scottohara.me/blog/2018/03/03/landmarks.html

For example, the documentation includes these bullets
> - Each page may have one banner landmark.
> - When a page contains nested document and/or application roles (e.g. typically through the use of iframe and frame elements), each document or application role may have one banner landmark.
> - If a page includes more than one banner landmark, each should have a unique label

Based on the explanation in the article, I interpret these points to mean that a page can only have more than one banner if additional banners are nested inside a `document` or `application`. 

> Looking through the design doc, it almost looks like this is used similar to an alert so I wonder if it should be treated similar with an aria-label.

We might want to consider not using aria-label on `<div>` elements that don't have an explicit role defined, like we currently do with the Alert component. The following article provides a really good explanation on that. But also, if visible text exists within an element, as in this case, the use of `aria-label` could override that text.
https://developer.paciellogroup.com/blog/2017/07/short-note-on-aria-label-aria-labelledby-and-aria-describedby/